### DISCUSSION TITLE
Still getting audio Premium ads

### DISCUSSION THREAD

**Discussion by Thechugg7** [SpotX-Official/SpotX#129] [Errors]
I seem to still have audio ads advertising premium every few songs.

**Comment by amd64fox** [comment]
https://github.com/amd64fox/SpotX/issues/new?assignees=&labels=%E2%9D%8C+bug&template=bug_report.yml

**Comment by Banjo65** [comment]
Hello 
I'm on Windows 11 and I receive advertising after about 5 minutes.
I tested the new theme version and also the complete theme.
every time Windows 11 updates my version of Spotify updates too.
and I don't have this problem on Windows 10
Install_New_theme.bat
Full installation
thank you for your reply!
